scripts/core/event.yml

```diff
@@ -448,7 +448,7 @@ class: $xCommandList
 name: AppendQueryKernelTimestamps
 details:
     - "The application must ensure the events are accessible by the device on which the command list was created."
-    - "The application must ensure the events were created from an event pool that was created using $X_EVENT_POOL_FLAG_KERNEL_TIMESTAMP flag."
+    - "The application must ensure the events were created from an event pool that was created using $X_EVENT_POOL_FLAG_KERNEL_TIMESTAMP or $X_EVENT_POOL_FLAG_KERNEL_MAPPED_TIMESTAMP flag."
```

Comment:
do we need to add note here that synchronized timestamps would not be available via this `appendQueryKernelTimestmps` ?

Reply:
@aravindksg Would it be better to disallow events from a `KERNEL_MAPPED` pool?

Reply:
@wdamon-intel yeah that might be a good restriction to add.